modules/operations/pages/monitoring/stream-audit-logs.adoc

```diff
@@ -0,0 +1,87 @@
+
+== Create an {product_name} topic for audit logs
+Audit log streaming requires a streaming tenant in the AWS `us-east-2` region.
+You can create a new tenant with the xref:astra-streaming:getting-started:index.adoc[{product_name} quickstart] or use an existing {product_name} tenant.
+
+. In the {link-astra-portal}, go to **Streaming**.
+. Select an existing tenant or xref:astra-streaming:getting-started:index.adoc#your-first-streaming-tenant[Create a Streaming Tenant] in AWS `us-east-2`.
+. Add a xref:astra-streaming:getting-started:index.adoc#add-a-namespace-to-hold-topics[namespace] and xref:astra-streaming:getting-started:index.adoc#a-topic-to-organize-messages[topic] to the tenant.
+. On the *Namespace and Topics* page, click the new topic, and then copy the topic's **Full Name**, such as `persistent://aws-us-east-2-mk/*NAMESPACE_NAME*/*TOPIC_NAME*`.
+. If necessary, create additional audit log topics, and then record the **Full Name** for each topic.
+Additional topics can help you organize audit logs by event type or other criteria.
+. In the {link-astra-portal}, go to **Streaming**, and then click your audit log streaming tenant.
+. On the *Connect* tab, click **Download client.conf**.
+. Send your topic's full address and the `client.conf` file to your DataStax representative. DataStax will complete the setup.
+
+== Configure the audit log with the DevOps API
+
+You can configure audit log streaming using the Astra DevOps API instead of contacting your DataStax representative.
+
+To do this, send a POST request to the Astra DevOps API telemetry endpoint. Include your topic’s full address and values from the `client.conf` file in the request. You'll need an **Organization Administrator** token to use this API.
+
+. Create the audit log streaming configuration.
++
+[source,curl,subs="verbatim,quotes"]
+----
+curl --request POST \
+  --url 'https://api.astra.datastax.com/v2/organizations/**ORG_ID**/telemetry/auditLogs' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer **APPLICATION_TOKEN**' \
+  --data '{
+  "pulsar": {
+    "endpoint": "pulsar+**BROKER_SERVICE_URL**",
+    "auth_strategy": "token",
+    "topic": "**TOPIC_FULL_NAME**",
+    "auth_name": "token",
+    "token": "**PULSAR_AUTHENTICATION_TOKEN**"
+  }
+}'
+----
++
+Response:
++
+[source,plain]
+----
+HTTP/1.1 202 Accepted
+----
++
+. Retrieve the audit log streaming configuration.
++
+[source,curl,subs="verbatim,quotes"]
+----
+curl --request GET \
+  --url 'https://api.astra.datastax.com/v2/organizations/**ORG_ID**/telemetry/auditLogs' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer **APPLICATION_TOKEN**' \
+  --data '{
+  "pulsar": {
+    "endpoint": "pulsar+**BROKER_SERVICE_URL**",
+    "auth_strategy": "token",
+    "topic": "**TOPIC_FULL_NAME**",
+    "auth_name": "token",
+    "token": "**PULSAR_AUTHENTICATION_TOKEN**"
+  }
+}'
```

Comment:
```suggestion
curl --request GET \
  --url 'https://api.astra.datastax.com/v2/organizations/**ORG_ID**/telemetry/auditLogs' \
  --header 'Authorization: Bearer **APPLICATION_TOKEN**'
```

Reply:
GET does not have a body.

https://docs.datastax.com/en/astra-api-docs/_attachments/devops-api/index.html?_gl=1*5e1j5c*_gcl_au*MjgzNDY4MjI0LjE3MjIyODk4NTM.#tag/Organization-Operations/operation/getTelemetryConfig

Reply:
Confirmed
{"pulsar":{"endpoint":"pulsar+ssl://pulsar-aws-useast2.streaming.datastax.com:6651","topic":"persistent://aws-us-east-2-mk-2/default/audit-log","auth_strategy":"token","token":"********","auth_name":"token"}}%

Reply:
Confirmed that it does or does not have a body? I don't see a request body in the reference, just a response example.